packages/docs/templates/footer.yaml.njk

```diff
@@ -249,7 +249,7 @@
 
 - id: footer_divider
   type: Divider
-- _ref: templates/built_with_lowdefy.yaml
+{# - _ref: templates/built_with_lowdefy.yaml #}
```

Comment:
Please add this back, the other footer will be removed once we add a license key

Reply:
Done - 9f29bb0645b59eb9a2958e6e3e1aaefb9ba0cea9